src/core/core-varlink.c

```diff
@@ -45,6 +46,42 @@ static bool user_match_lookup_parameters(LookupParameters *p, const char *name,
         return true;
 }
 
+static int vl_method_get_oomd_enabled_cg_paths(Varlink *link, JsonVariant *parameters, VarlinkMethodFlags flags, void *userdata) {
+        _cleanup_(json_variant_unrefp) JsonVariant *v = NULL;
+        _cleanup_strv_free_ char **p = NULL;
+        Manager *m = userdata;
+        CGroupContext *c;
+        Unit *u;
+        int r;
+
+        assert(link);
+        assert(m);
+
+        if (json_variant_elements(parameters) > 0)
+                return varlink_error_invalid_parameter(link, parameters);
+
+        /* Only slices support EnableOomdKill= right now */
+        LIST_FOREACH(units_by_type, u, m->units_by_type[UNIT_SLICE]) {
+                if (unit_active_state(u) == UNIT_ACTIVE) {
```

Comment:
doesn't matter much, but in the interest of keeping the indentation level small, i'd invert this check:

```c
if (unit_active_state(u) != UNIT_ACTIVE)
        continue;
```